.circleci/config.yml

```
@@ -350,17 +350,17 @@ workflows:
           - back_behat_legacy:
                 requires:
                     - build_dev
-pull_up:
-  jobs:
-    - pull-up/pull-up:
-        from: "4.0"
-        into: "5.0"
-        branch_prefix: "40_to_50"
-        checkout_ours: 'src/Akeneo/Platform/CommunityVersion.php **/translations/*.yml'
-        github_token: MICHEL_TAG_TOKEN
-        github_username: "micheltag"
-        slack_webhook: "${SLACK_URL_PULL_UP}"
-        filters:
-          branches:
-            only:
-              - "4.0"
+  pull_up:
+    jobs:
+      - pull-up/pull-up:
+          from: "4.0"
+          into: "5.0"
+          branch_prefix: "40_to_50"
+          checkout_ours: 'src/Akeneo/Platform/CommunityVersion.php **/translations/*.yml'
+          github_token: MICHEL_TAG_TOKEN
+          github_username: "micheltag"
+          slack_webhook: "${SLACK_URL_PULL_UP}"
+          filters:
+            branches:
+              only:
+                - "4.0"
```

I would test it with a temporary commit by filtering on your current branch `fix-pull-up` instead of 4.0. Otherwise, you don't really know if it works as expected.

```suggestion
                - "fix-pull-up"
```

---

It works!
https://app.circleci.com/pipelines/github/akeneo/pim-community-dev/21929/workflows/81a81c99-61e8-42e3-9a8a-1f52591a6e98/jobs/90816